packages/workbox-streams/isSupported.mjs

```diff
@@ -19,15 +19,20 @@ import './_version.mjs';
  * This is a utility method that determines whether the current browser supports
  * the features required to create streamed responses. Currently, it checks if
  * [`ReadableStream`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/ReadableStream)
- * is available.
+ * can be created.
  *
  * @return {boolean} `true`, if the current browser meets the requirements for
  * streaming responses, and `false` otherwise.
  *
  * @memberof workbox.streams
  */
 function isSupported() {
-  return 'ReadableStream' in self;
+  try {
```

Comment:
It's probably worth adding an explanation or linking to the bug explaining why this is needed.

Also, does this get invoked more than once in a stream strategy context? If so maybe it's worth caching the `isSupported` value?